tst/testmanuals.g

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
+#W  testmanuals.g
+##
+##  <#GAPDoc Label="[1]{teststandard.g}">
+##  <#/GAPDoc>
+##
+
+# This code extracts the examples from manuals chapter-wise and
+# stores this in a workspace.
+
+Print("Extracting manual examples...\n");
+Read(Filename(DirectoriesLibrary("doc/ref"), "makedocreldata.g"));
+
+GAPInfo.ManualDataRef.pathtodoc := DirectoriesLibrary("doc/ref");
+GAPInfo.ManualDataRef.pathtoroot := DirectoriesLibrary("");
+
+exsref := ExtractExamples(
+    GAPInfo.ManualDataRef.pathtodoc,
+    GAPInfo.ManualDataRef.main,
+    GAPInfo.ManualDataRef.files,
+    "Chapter" );
+
+# RS := rec(changeSources := true);
+# WS := rec(compareFunction := "uptowhitespace");
+# WSRS := rec(changeSources := true, compareFunction := "uptowhitespace");
+
+WriteExamplesTst := function(directory)
+    local ch, chname, i, a, output;
+    for i in [1..Length(exsref)] do
+        ch := exsref[i];
+        chname := STRINGIFY("chapter", i, ".tst");
+        output := OutputTextFile( Concatenation(directory, "/", chname), false );
+        SetPrintFormattingStatus( output, false );
+
+        PrintTo(output, "####  Reference manual, Chapter ",i,"  ####\n",
+                "gap> START_TEST(\"", chname, "\");\n");
+        for a in ch do
+            AppendTo(output, "\n#LOC# ", a[2], a[1]);
```

Comment:
Another minor nitpick: You switch between `PrintTo` and `AppendTo`. I think they do the same for streams anyway, but using both might confuse somebody who doesn't know that...